apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -117,11 +118,21 @@ async function canManageTickets (args) {
         let organizationId
 
         if (operation === 'create') {
+            if (isBulkRequest) return false
             organizationId = get(originalInput, ['organization', 'connect', 'id'])
         } else if (operation === 'update') {
-            if (!itemId) return false
-            const ticket = await getById('Ticket', itemId)
-            organizationId = get(ticket, 'organization', null)
+            if (isBulkRequest) {
```

Comment:
are there any other checks for singular update case?

Reply:
These cases are below. There are cases when updating and creating one ticket - the property must belong to the same organization for which the ticket is created